arduino-ide-extension/src/browser/dialogs/ide-updater/ide-updater-dialog.tsx

```diff
@@ -165,6 +165,38 @@ export class IDEUpdaterDialog extends ReactDialog<UpdateInfo | undefined> {
     goToDownloadPageButton.focus();
   }
 
+  private appendDonateFooter() {
+    const footer = document.createElement('div');
+    footer.classList.add('ide-updater-dialog--footer');
+
+    footer.innerText = nls.localize(
+      'arduino/ide-updater/donateText',
+      'Open source is love'
```

Comment:
Would it be possible to do something like this here:

```ts
'Open source is love, {0}'
```

where `{0}` is the placeholder in the template for the donation link?

The reason this would be better is because the current code imposes the grammatical structure `<arduino/ide-updater/donateText>, <arduino/ide-updater/donateLinkIconTitle>`, which won't necessarily be appropriate for all locales. The template approach I describe above allows the translators to adjust the structure of the text as is appropriate for the locale.

The internationalization system supports this template syntax, no problem. For example, you can see it used here:

https://github.com/arduino/arduino-ide/blob/c09b5f718a5a14c420c1649ac5bf117d8fd68d9e/arduino-ide-extension/src/browser/boards/boards-auto-installer.ts#L161-L167

but perhaps it is more challenging in this case where `arduino/ide-updater/donateLinkText` must be linkified.

Reply:
Unfortunately, the internationalization system in Theia only supports replacing primitive values in translation strings; custom elements are not supported. However, I’ve added custom code to allow the anchor element to render at the position defined by the wildcard. This solution currently works only with this specific string format, but we may extend it in the future if this use case becomes more frequent.